### PR TITLE
Parser for show lisp site

### PR DESCRIPTION
**Pull request by reustaquio22**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_lisp.py [hunk 1]
```diff
@@ -3267,3 +3267,148 @@ def cli(self, service, instance_id, output=None):
...
+
+
+# ===================
+# Schema for:
+#  * 'show lisp site'
+# ===================
+class ShowLispSiteSchema(MetaParser):
+    """Schema for show lisp site."""
+
+    schema = {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/show_lisp.py:3279]: schema looks good

File: changelog/undistributed.rst [hunk 2]
```diff
@@ -10,6 +10,9 @@
 --------------------------------------------------------------------------------
 
 * IOSXE
+    * added showstackwisevirtual
```
Comment (LenzDong) [inline changelog/undistributed.rst:13]: Please remove this `showstackwisevirtual`, only add the change related to this pull request

Comment (reustaquio22) [reply changelog/undistributed.rst]: @LenzDong, This was removed.

**Comment by badalsoyantar** [comment]
Resolve conflicts

**Comment by reustaquio22** [comment]
@badalsoyantar, Conflicts were resolved

**Pull request merged**